### PR TITLE
Work for release 0.23.x

### PR DESCRIPTION
**Pull request by reshmabidikar**

### CODE REVIEW

File: docker/docker-compose.ci.mysql.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+version: '3.8'
+
+services:
+  killbill:
+    network_mode: host
+    image: killbill/killbill:0.23.0-9116fa4-SNAPSHOT
```
Comment (pierre) [inline docker/docker-compose.ci.mysql.yml:23]: Could upgrade to 0.24. now :-)

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -4,40 +4,20 @@ on:
...
+    secrets:
+      extra-env: '{ "BRAINTREE_ENVIRONMENT": "sandbox", "BRAINTREE_MERCHANT_ID": "${{ secrets.BT_MERCHANT_ID }}", "BRAINTREE_PRIVATE_KEY": "${{ secrets.BT_PRIVATE_KEY }}", "BRAINTREE_PUBLIC_KEY": "${{ secrets.BT_PUBLIC_KEY }}" }'
+
+  integration_test:
+    uses: killbill/gh-actions-shared/.github/workflows/integration_test.yml@main
+    with:
+      plugin_name: 'braintree'
+      integration_tests_goal: 'test:plugins:braintree'
+      ddl_file: 'src/main/resources/ddl.sql'
+      integration_tests_ref: 'refs/heads/work-for-release-0.23.x'
```
Comment (pierre) [inline .github/workflows/ci.yml:21]: Maybe we should merge the branch and point all of these CI files to master now.

**Review by pierre — APPROVED** [review]

File: docker/docker-compose.ci.mysql.yml (hunk 1 above)
Comment (reshmabidikar) [reply docker/docker-compose.ci.mysql.yml:23]: Yup, I will do this after merging to master 👍

File: .github/workflows/ci.yml (hunk 2 above)
Comment (reshmabidikar) [reply .github/workflows/ci.yml:21]: Do you mean merging the `work-for-release-0.23.x` branch of the integration tests repo to `master`? I've already done this and made changes in the other plugins to point to `master`. For example, take a look at the [killbill-stripe/ci.yml](https://github.com/killbill/killbill-stripe-plugin/blob/ec31abcfa6ef3a95a877ce05d6f9db370c17471d/.github/workflows/ci.yml#L21) file.

**Pull request merged**